src/main/java/com/twilio/sdk/resource/InstanceResource.java

```diff
@@ -153,6 +153,16 @@ public Integer getPropertyAsInteger(String name) {
 	}
 
 	/**
+	 * Gets the property as a Boolean value.
+	 *
+	 * @param name property name
+	 * @return Boolean value
+	 */
+	public Boolean getPropertyAsBoolean(String name) {
+		return Boolean.valueOf(getProperty(name));
```

Comment:
@senthgit those don't test the scenario I mentioned because you are mocking getProperty. I mentioned that : 
`I'm pretty sure getProperty will try to cast it to a string and explode if it cannot.`

Which means you should test by either: 
1- inserting a boolean object into the `properties` map of an instance.
2- converting a json response that contains a boolean value and attempt to extract that value.